uwrt_mars_rover_hw/drivers/roboteq_driver/mbs/roboteq_control_mode_switching.mbs

```diff
@@ -0,0 +1,18 @@
+print("Starting roboteq drivetrain control mode switching ... \r")
+current_control_mode_1 = getvalue(_MMOD, 1)
+current_control_mode_2 = getvalue(_MMOD, 2)
+
+top: 
+desired_control_mode = getvalue(_VAR, 9) ' Read desired control mode from ROS
+
+if(current_control_mode_1 = current_control_mode_1 And desired_control_mode <> current_control_mode_1) Then
+	current_control_mode_1 = setconfig(_MMOD, 1, desired_control_mode) ' switch control modes

```

Comment:
setconfig doesnt return anything, so u need to either assign current_control_mode_1 and current_control_mode_2 to desired_control_mode or get their values from `GetConfig(_MMOD, 1 or 2)`